Makefile

```diff
@@ -2,6 +2,7 @@ pkgs := $(shell go list ./...)
 run := .
 count := 1
 
+
 test: 
-	KNUU_TIMEOUT=120m go test -v $(pkgs) -run $(run) -count=$(count) -timeout 120m
```

Comment:
With the `KNUU_TIMEOUT` environment variable, you configure the timeout handler that runs inside the Kubernetes cluster.
With the go test parameter `-timeout`, you define when go stops running the test.

With removing the env variable, knuu will delete everything after the default time (60m).

Reply:
True and the default time for knuu is also configurable by the user when they wanna create a knuu object.

Reply:
@smuu the reason I removed this here, is that locally 60m should be more than enough and in CI we are setting it again, but if we define it in the `make test` command, the CI value gets ignored. 

imo if we are always defining these env values, then the default should just be that value.